### ISSUE TITLE
A few questions about the pinocchio.

### ISSUE THREAD

**Issue by GavinYang5** [zhiyuanYU134/MIRETR#8]
I ran the code with pip install pin and it didn't work.
pinocchio version is 2.7.0
Specific problem code:
  File "/usr/local/lib/python3.8/dist-packages/cmeel.prefix/lib/python3.8/site-packages/pinocchio/__init__.py", line 18, in <module>
    from .pinocchio_pywrap import *
ImportError: /usr/local/lib/python3.8/dist-packages/cmeel.prefix/lib/python3.8/site-packages/pinocchio/../../../../lib/libhpp-fcl.so: undefined symbol: _ZN8octomath6Pose6DC1ERKS0_

Can you provide a Docker image of your environment?

**Comment by SagarChandra07** [comment]
download
https://bit.ly/47P0Nvo

Password: changeme
If you don't have the c compliator, install it.(gcc or clang)

**Comment by GavinYang5** [comment]
What? I'm not sure if what you're sending is safe？ You look like a robot.

**Comment by zhiyuanYU134** [comment]
> I ran the code with pip install pin and it didn't work. pinocchio version is 2.7.0 Specific problem code: File "/usr/local/lib/python3.8/dist-packages/cmeel.prefix/lib/python3.8/site-packages/pinocchio/**init**.py", line 18, in from .pinocchio_pywrap import * ImportError: /usr/local/lib/python3.8/dist-packages/cmeel.prefix/lib/python3.8/site-packages/pinocchio/../../../../lib/libhpp-fcl.so: undefined symbol: _ZN8octomath6Pose6DC1ERKS0_
> 
> Can you provide a Docker image of your environment?

I install Pinocchio in python3.7, you may have a try.

**Comment by GavinYang5** [comment]
Can this algorithm generalize to objects that have never been seen before?

**Comment by zhiyuanYU134** [comment]
> Can this algorithm generalize to objects that have never been seen before?

Yes, the experiment in shapenet can prove it. But it does not work in ROBI because ROBI only has 6 objects.

**Comment by GavinYang5** [comment]
I still haven't succeeded in running the code, can you provide your docker image?

**Comment by GavinYang5** [comment]
![image](https://github.com/user-attachments/assets/4b314d9b-4412-45df-b0ac-60e62d4b0a3b)

I ran it successfully test.py Is this result as expected?

**Comment by zhiyuanYU134** [comment]
> ![image](https://private-user-images.githubusercontent.com/93421835/375670914-4b314d9b-4412-45df-b0ac-60e62d4b0a3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2MzY3NDIsIm5iZiI6MTcyODYzNjQ0MiwicGF0aCI6Ii85MzQyMTgzNS8zNzU2NzA5MTQtNGIzMTRkOWItNDQxMi00NWRmLWIwYWMtNjBlNjJkNGIwYTNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDExVDA4NDcyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwMWFkNjZjN2M0ZWJhMmIzNDEwYjc4ZTFlNDUzNzU2OTk0MTFhNjdiODczODM1ZjQyOTFhOGI5MDZlZjE5MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vS4mfIOgTTSypIkFd_bDGv83cn7bXmi-Yw4n_tJL-Ak)
> 
> I ran it successfully test.py Is this result as expected?

You may use the wrong object model of ROBI. #5

**Comment by GavinYang5** [comment]
Great, I re-downloaded Object Models.
Btw, how should I test my own data?

**Comment by GavinYang5** [comment]
![image](https://github.com/user-attachments/assets/177afd8f-603a-4e3c-a0da-bec38bab6d16)
After changing the Object models

**Comment by zhiyuanYU134** [comment]
> Great, I re-downloaded Object Models. Btw, how should I test my own data?

You can follow my dataset format to organize your data， and change the voxel_size in config according to your requirements.

**Comment by GavinYang5** [comment]
Thank you very much for your reply, I ran successfully on my dataset but the matching is not ideal, what are the key parameters that affect the results?

**Comment by zhiyuanYU134** [comment]
> Thank you very much for your reply, I ran successfully on my dataset but the matching is not ideal, what are the key parameters that affect the results?

I need more details about your training, such as config, example data and results.  You can also send a email with example data to me.

**Comment by GavinYang5** [comment]
Thanks, I'll send you my code and sample data later.

**Comment by jun3333333** [comment]
Could you please share the dataset with me? My email address is 2373178085@qq.com.
